### DISCUSSION TITLE
Why isn't my trained NER model not able to assign 'O' tag to a sample text?

### DISCUSSION THREAD

**Discussion by Parth-tech** [explosion/spaCy#10438] [Help: Coding & Implementations] [answered]
I had some doubts regarding my trained model and as I am new to spacy, I need some assistance.....
I followed the below steps:

1. I have multiple training files in **_.TSV_** format. I directly converted them into **_.SPACY_** format. (The **_TSV file_** has following the entities: _'GENE', 'DATE', 'QUANTITY', 'DISEASE', 'BODY_PART', 'PERSON', 'O', 'CHEMICAL',  'CELL', 'SPECIES',  'PROCEDURE',  'PLACE', 'PROCESS', 'PHENOMENON', 'VIRUS'_ and it in the **_IOB format_**). I have attached a sample TSV file. (here O means that the word is not important)

2. Then I downloaded a config file, I had chose **_CPU_** as **_Hardware_** and for **_Components_** I chose **_NER_**.

3.  I trained the model but my testing file was subset of training file so I got unreasonably high score. But the problem is in the **_Meta.json_** file I cant see the **_O tag_** under **_NER_** but I used it in the training file.

[10.txt](https://github.com/explosion/spaCy/files/8191332/10.txt)
![image](https://user-images.githubusercontent.com/56110372/156897834-f3a172b2-8ba6-42c9-b353-42eb5ec7b01b.png)

**Comment by polm** [comment, marked as answer]
Please copy and paste text as text, not as images. 

> I trained the model but my testing file was subset of training file so I got unreasonably high score. But the problem is in the Meta.json file I cant see the O tag under NER but I used it in the training file.

The O tag is always present so it's not included in the list, it's automatically added, that's all.

**Comment by Parth-tech** [comment]
Thank you!
